dace/transformation/dataflow/copy_to_map.py

```diff
@@ -62,31 +72,69 @@ def delinearize_linearize(self, desc: data.Array, copy_shape: Tuple[symbolic.Sym
         return subsets.Range([(ind, ind, 1) for ind in cur_index])
 
     def apply(self, state: SDFGState, sdfg: SDFG):
-        adesc = self.a.desc(sdfg)
-        bdesc = self.b.desc(sdfg)
-        edge = state.edges_between(self.a, self.b)[0]
+        avnode = self.a
+        av = avnode.data
+        adesc = avnode.desc(sdfg)
+        bvnode = self.b
+        bv = bvnode.data
+        bdesc = bvnode.desc(sdfg)
+
+        edge = state.edges_between(avnode, bvnode)[0]
+        src_subset = edge.data.get_src_subset(edge, state)
```

Comment:
Do you think we may need to call the `try_initialize` method before attempting to get src/dst subsets here?

Reply:
I am actually not sure.
I was hit by this a few time in the past so I am in favour of keeping it.
However, the main reason why this function is used (I think I "discovered" it here) is because even in the original code it is used (see old line 70).
For that reason I decided to keep it.
But if you think we should remove that I will not object.

Reply:
I mean, should we also add a call to the method that tries to initialize the edge/meet to ensure that the src/dst subsets are not None when they are not supposed to.